model/database/common.yaml

```diff
@@ -30,3 +30,32 @@ groups:
           generic one, the original exception SHOULD be preferred.
 
           Instrumentations SHOULD document how `error.type` is populated.
+
+  - id: attributes.db.cosmosdb.minimal
```

Comment:
nit: this group is only used in cosmos metrics, please consider moving it to the `cosmosdb-metrics.yaml` file

Reply:
here https://github.com/open-telemetry/semantic-conventions/pull/1438#discussion_r1800271314, you suggested me to put this group to `common.yaml`